packages/subgraph-service/contracts/SubgraphService.sol

```diff
@@ -372,9 +377,8 @@ contract SubgraphService is
     /**
      * @notice See {ISubgraphService.setStakeToFeesRatio}
      */
-    function setStakeToFeesRatio(uint256 stakeToFeesRatio_) external override onlyOwner {
-        stakeToFeesRatio = stakeToFeesRatio_;
-        emit StakeToFeesRatioSet(stakeToFeesRatio_);
+    function setStakeToFeesRatio(uint256 stakeToFeesRatio) external override onlyOwner {
```

Comment:
I'd keep the `_` here just in case

Reply:
yeah, I was on the fence but makes sense, updated!